src/popup/locales/cn.json

```diff
@@ -1,905 +1,990 @@
-      "section9Title": "9. INTELLECTUAL PROPERTY RIGHTS",
-      "section9Content": "USERS understand and accept ​the application can be periodically modified, updated, maintained or technically supported, which could to some extent influence the use of the Wallet​, the visual representation of ​the application ​or could result in malfunction. In the latter case, if ​You as a ​USER notice any change in the funds, settings, accounts or any other information on the application, please superherowallet@protonmail.com​ for assistance.",
-      "section10Title": "10. MODIFICATIONS",
-      "section10Content": "The legislation and jurisdiction of Liechtenstein is applied for the issues outstanding the present ​TERMS​. The invalidity of one or more clauses/ provisions in the ​TERMS does not lead to the invalidity of the whole agreement. \n The Privacy Policy is an inseparable part of the present ​TERMS and should be considered as a complementary document with compulsory character. Unless explicitly stated otherwise in the Privacy Policy, all rules from the ​TERMS​ apply respectively.",
-      "section11Title": "11. FINAL PROVISIONS"
+      "superhero-we-us-with-quotes": "(“超级英雄”, “我们”, “我们”",
+      "our-with-quotes": "“我们的”)",
+      "section1Title": "1. 定义",
+      "section1Content": "1.1.Superhero.com(Superhero.com ”, “我们”, “我们”，“我们的”或“公司”)成立的是家公司，超级英雄钱包相关问题的邮箱联系地址是: superherowallet@protonmail.com。超级英雄钱包由Superhero.com公司创造。 \n 1.2. ​„使用者“或者„你“ 代表下载并使用超级英雄钱包网站扩展的自然人或法人。\n 1.3. „​区块链“代表记录清单，也称为区块，区块间以密码学方法相连。​这个技术允许大量的人对数据进行获取并达成共识，之后数据会被永久地记录下来了，没有中央权威的存在。用户可以在aeternity区块链上获取，创造，保存信息。分散式、分布式和公共的数字账本用于在验证器的帮助下以信息块的形式记录交易，这些信息是不能被逆向更改的。\n 1.4。“超级英雄钱包”(“钱包”或“应用程序”)是一个网站扩展，它允许支持者向内容创建者、事业、慈善机构、团体或普通个人发送价值，作为对贡献准确信息、相关见解和高质量内容的奖励。网站扩展是一个软件，作为一个附添加到网络浏览器Firefox并为浏览器提供附加功能。该应用程序可以在https://addons.mozilla.org/en-us/firefox/addon/superher-wallet-extension/获取。捐款和评论的结果发布在https://superhero.com/网站上。\n 1.5。“验证者”是独立的自然人或实体，他们在任何方面都不属于Superhero.com的一部分。它们在系统中的功能包括模块的组成和在aeternal网络中传播这些模块。1.6 \n。“帐户地址”是用户公钥的散列版本，用于在应用程序中识别用户。1.7 \n。“密钥”表示唯一相关的用于加密和解密信息的随机数私钥和公钥。它们在系统中的功能包括区块的组成和在aeternity网络中对区块的广播。 \n 1.6. “帐户地址”是用户公钥的hash版本，用于在应用程序中识别用户。 \n 1.7. “密钥”代表唯一相关的用于加密和解密信息的随机数私钥和公钥。",
```

Comment:
There are duplicates of 1.6 and 1.7 - can you double-check and verify which one is correct?